Dockerfile

```diff
@@ -1,12 +1,16 @@
 FROM jupyter/datascience-notebook
 
-MAINTAINER Pierre Bellec <pierre.bellec@gmail.com>
+LABEL maintainer="Pierre Bellec <pierre.bellec@gmail.com>"
 
 USER jovyan
 
-RUN pip install nilearn
+# Copying the repository inside the container
+COPY . /home/jovyan/work
 
-# Get the files of the project
-RUN wget https://github.com/brainhack101/introML/archive/0.1.zip
-RUN unzip 0.1.zip
-RUN rm 0.1.zip
+# Instaling the requirements
+RUN pip install nilearn \
+	osfclient
+
+# Downloading the data (osf data with a fake email)
```

Comment:
Is there a reason to use an email and password, here ? Generally I'm a little twitchy about putting passwords in a Dockerfile, but since this is a publicly accessible file I'm just not sure why we need to do it.

Reply:
so I created a fake mail with a fake password, of course I did`t put a "true" passwd here haha